src/hivpy/hiv_status.py

```diff
@@ -222,6 +231,8 @@ def update_HIV_status(self, population: Population):
                                                  (col.DATE_HIV_INFECTION, operator.is_, None)])
```

Comment:
Need to use equals here to make the infection date work as intended, otherwise this updates every infected person's infection date to the current date every time step.
```suggestion
                                                 (col.DATE_HIV_INFECTION, operator.eq, None)])
```